src/ergw_aaa_nasreq.erl

```diff
@@ -541,3 +548,58 @@ create_STR(Session, Opts) ->
 
 get_state_atom(#state{state = State}) ->
     State.
+
+handle_common_request(Command, #{'Session-Id' := SessionId} = Avps, {_PeerRef, Caps}) ->
+    {Result, ReplyAvps0} =
+	case ergw_aaa_session_reg:lookup(SessionId) of
+	    Session when is_pid(Session) ->
+		ergw_aaa_session:request(Session, ?MODULE, {?API, Command}, Avps);
+	    _ ->
+		{{error, unknown_session}, #{}}
+	end,
+
+    #diameter_caps{origin_host = {OH,_},
+		   origin_realm = {OR,_},
+		   origin_state_id = {OSid, _}} = Caps,
+
+    ReplyAvps1 = filter_reply_avps(Command, ReplyAvps0),
+    ReplyAvps2 =
+	ReplyAvps1#{'Origin-Host' => OH,
+		    'Origin-Realm' => OR,
+		    'Origin-State-Id' => OSid,
+		    'Session-Id' => SessionId},
+    ReplyCode = diameter_reply_code(Command),
+    ReplyAvps = diameter_reply_avps(Result, ReplyAvps2),
```

Comment:
The name is too generic. The function add result code.

Reply:
Yes it is, but all other handlers contain the same mistake. Lets leave it as is in this PR and fix all handlers in one go.

Reply:
> I fact the whole function is a direct copy from the other modules. So far nasreq, ro and gx all use the same common request handler. Moving them all into the generic diameter support module would be a good follow-up change.

Yes I was see it too and want to provide separate PR for small refactoring of it